src/core/execute.h

```diff
@@ -108,6 +108,21 @@ struct ExecRuntime {
         int netns_storage_socket[2];
 };
 
+typedef enum ExecDirectoryType {
+        EXEC_DIRECTORY_RUNTIME = 0,
+        EXEC_DIRECTORY_STATE,
+        EXEC_DIRECTORY_CACHE,
+        EXEC_DIRECTORY_LOGS,
+        EXEC_DIRECTORY_CONF,
```

Comment:
i think this should be spelled out, given tha the configuration settings are spelled out too: EXEC_DIRECTORY_CONF → EXEC_DIRECTORY_CONFIGURATION. I am not a fan of needless abbreviations...